cli/command/image/load.go

```diff
@@ -35,12 +40,13 @@ func NewLoadCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags.StringVarP(&opts.input, "input", "i", "", "Read from tar archive file, instead of STDIN")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress the load output")
+	flags.StringVarP(&opts.name, "name", "n", "", "Name to use when loading OCI image layout tar archive")
+	flags.StringSliceVar(&opts.refs, "ref", []string{}, "References to use when loading an OCI image layout tar archive")
```

Comment:
OCI is a bit different - a `ref` flag here translates an OCI reference (essentially a tag) e.g. `latest` to a docker named reference.
The thing is OCI doesn't have naming, so what we get is just a tarball containing multiple references (potentially, even different images).

Not sure how to address this comment about `docker build -t` - it's something a bit different to me.